corehq/apps/userreports/rebuild_utils.py

```diff
@@ -135,9 +135,12 @@ def _get_indexes_to_change(raw_diffs, table_names):
             indexes_to_change.append(index_diffs[0])
         else:
             # tests when alembic tries to remove/add the same index
```

Comment:
Pretty sure this comment was my own, and I don't think it's that great. Maybe change it to:

`# do nothing if alembic attempts to remove and add an index on the same column`